README.template.md

```diff
@@ -59,4 +59,12 @@ Coming soon!
 
 Coming soon!
 
+### Running Tests
```

Comment:
These instructions are relevant to Momento devs but probably not to most OSS users.  So we should create a `CONTRIBUTING.md` file and put them in there instead of in the main README.

Take a look at the other SDK repos and see if we have an example of a `CONTRIBUTING.md` anywhere yet.  Ultimately we will want to have one in every SDK repo and we will want to make them somewhat consistent, but I'm not sure if we've really established a pattern for that yet.  If not, you can just get us started :)

I would recommend having two different sections in the CONTRIBUTING.md: one for how to build the docker image, and one for how to run things.  Our goal will be for devs to only need to build the docker image once (or very rarely), and then they should only need to do variations on `docker run` once they have the image.